addons/hr_work_entry_contract/wizard/hr_work_entry_regeneration_wizard.py

```diff
@@ -98,7 +101,9 @@ def regenerate_work_entries(self):
             ('date_start', '<=', date_to),
             ('state', '!=', 'validated')])
 
-        work_entries.write({'active': False})
+        write_vals = {'active': False}
```

Comment:
Is there a reason why you are not adding `active` to `fields_to_nullify`? :thinking:

Reply:
No 😄